lua/coq/config.lua

```diff
@@ -0,0 +1,54 @@
+local config = {
```

Comment:
Sorry about deleting your work.

`coq` has it's own config merge already, and the default values are also defined in a spec file.

so this is redundant

Reply:
https://github.com/ms-jpq/coq_nvim/blob/coq/config/defaults.yml

Reply:
It's cool as long as user can override them using setup(). It's a best practice pretty much all lua plugins use.